app/react-components/.eslintrc

```diff
@@ -10,6 +10,7 @@
     "fp/no-mutation": 0,
     "fp/no-mutating-methods": 0,
     "fp/no-this": 0,
+    "jsx-a11y/no-static-element-interactions": 0,
```

Comment:
Yeah okay, i'll put this back and replace it with `<UnstyledButton />` or something